lib/amazonka/amazonka.cabal

```diff
@@ -95,7 +96,10 @@ library
     Amazonka.Data, Amazonka.Types, Amazonka.Bytes, Amazonka.Endpoint, Amazonka.Crypto
 
   build-depends:
+    , Glob
```

Comment:
Please add bounds here. From looking at the changelog I think `>= 0.9 && <0.11` should be safe. Then `cabal-fmt --inplace` the cabal file.